bazel/external/wee8-ppc64le.patch

```diff
@@ -0,0 +1,47 @@
+--- wee8/BUILD.gn
++++ wee8/BUILD.gn
+@@ -1111,8 +1111,16 @@ config("toolchain") {
+ 
+   if (is_clang) {
+     cflags += [ "-Wmissing-field-initializers" ]
++    if (v8_current_cpu == "ppc64le") {
++      cflags += [
++        "-fno-integrated-as",
++        "-Wno-unreachable-code",
++        "-Wno-invalid-offsetof",
++      ]
++      ldflags = [ "-fuse-ld=lld" ]
++    }
+ 
+-    if (v8_current_cpu != "mips" && v8_current_cpu != "mipsel") {
++    if (v8_current_cpu != "mips" && v8_current_cpu != "mipsel" && v8_current_cpu != "ppc64") {
```

Comment:
should this use `ppc64le`?

Reply:
pcc64le should be replace with ppc64

Reply:
@jwendell  changes done in latest commit , can you please review